src/Week1/HW01.hs

```diff
@@ -37,15 +58,20 @@ doubleEveryOther lst = helper 0 lst []
 
 -- Calculate the sum of all the digits in every Integer.
 sumDigits :: [Integer] -> Integer
-sumDigits lst = sum lst 
+-- Performs in one pass
+sumDigits = foldr (helper) 0
+    where
+        helper :: Integer -> (Integer -> Integer)
+        helper n = 
+            toDigits n & sum & (+)
 
 
 -- Exercise 5 -----------------------------------------
 
 -- Validate a credit card number using the above functions.
 
 luhn :: Integer -> Bool
-luhn n = toRevDigits n & doubleEveryOther & sumDigits & (`mod` 10)  & (==10)
+luhn n = toRevDigits n & doubleEveryOther & sumDigits & (`mod` 10)  & (==0)
```

Comment:
As discussed on the call, using `.` and `$` is more idiomatic haskell.
You can leave these as they are but let's use . and and $ in future code.